python/nano/src/bigdl/nano/pytorch/inference/optimizer.py

```diff
@@ -160,6 +165,8 @@ def optimize(self, model: nn.Module,
 
         result_map: Dict[str, Dict] = {}
 
+        model.eval()  # change model to eval state
```

Comment:
Can you first record model's status, and restore it after we are done?

Reply:
Yes. After update, now I record model's status by `model.training` at the beginning of `optimize`, and restore it at the end of `optimize`.
So if a model is in training status, after `optimize`, it is still in training status, but all accelarated models will be in eval status always.